tests/_data/_serialize/test_memory_mapped_sequence.py

```diff
@@ -8,105 +8,191 @@
 import pickle
 from pathlib import Path
 
-import pyarrow as pa  # type: ignore
 import pytest
-from pyarrow import ipc
 
 from lightly_train._data._serialize import memory_mapped_sequence
-from lightly_train._data._serialize.memory_mapped_sequence import MemoryMappedSequence
+from lightly_train._data._serialize.memory_mapped_sequence import (
+    MemoryMappedSequence,
+)
 
 
 class TestMemoryMappedSequence:
     def test_index(self, tmp_path: Path) -> None:
-        memory_mapped_sequence.write_filenames_to_file(
-            filenames=["image1.jpg", "image2.jpg", "image3.jpg"],
+        image_dir = tmp_path / "images"
+        mask_dir = tmp_path / "masks"
+        image_dir.mkdir()
+        mask_dir.mkdir()
```

Comment:
You don't need to create the directory or use `tmp_path` as the images/masks are just strings and the underlying files are never accessed.